src/components/flowchart/index.js

```diff
@@ -354,15 +354,15 @@ export class FlowChart extends Component {
    * @param {Object} node Datum for a single node
    */
   handleNodeClick = node => {
-    this.props.onToggleNodeClicked(node.id);
+    this.props.loadNodeData(node.id);
```

Comment:
This should be `this.props.onLoadNodeData(node.id);`. It currently throws an error.
![image](https://user-images.githubusercontent.com/1155816/97995147-0cd28300-1dde-11eb-849d-52da3298fe33.png)

Reply:
great spot, thank you!